### ISSUE TITLE
Rep place

### ISSUE THREAD

**Issue by stefanwalther** [assemble/generator-verb#11]
Hi,

this should probably also moved to https://github.com/verbose, right?

Regards
Stefan

**Comment by jonschlinkert** [comment]
yeah, we'll move verb and this repo over to verbose once I get this refactor done.
